app/forms/delivery_partners/choose_organisation_form.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module DeliveryPartners
+  class ChooseOrganisationForm
+    include ActiveModel::Model
+    include ActiveModel::Attributes
+
+    attribute :user
+    attribute :delivery_partner_id
+
+    validates :delivery_partner_id, inclusion: { in: :valid_ids }
+
+    def delivery_partner_options
+      @delivery_partner_options ||= user.delivery_partners.each_with_object({}) do |ab, sum|
+        sum[ab.id] = ab.name
+      end
+    end
+
+    def delivery_partner
+      return if delivery_partner_id.blank?
+
+      user.delivery_partners.find(delivery_partner_id)
+    end
+
+    def only_one
```

Comment:
predicate?

Reply:
@asmega 
As we're setting the `delivery_partner_id` here, I've removed the `?`